app/controllers/movies_controller.rb

```diff
@@ -79,17 +70,13 @@ def update
   # DELETE /movies/1
   # DELETE /movies/1.json
   def destroy
-    @movie.delete_all_reviews
```

Comment:
I added `has_many :reviews, dependent: :destroy` in `Movie`, so when we call `@movie.destroy`, the associate reviews also destroyed [More](http://apidock.com/rails/v4.2.1/ActiveRecord/Associations/ClassMethods/has_many)